rules/windows/process_creation/proc_creation_win_renamed_pingcastle.yml

```diff
@@ -0,0 +1,32 @@
+title: Renamed PingCastle Binary Execution
+id: 2433a154-bb3d-42e4-86c3-a26bdac91c45
+status: experimental
+description: Detects the execution of a renamed "PingCastle" binary based on the PE metadata fields.
+references:
+    - https://thedfirreport.com/2023/10/30/netsupport-intrusion-results-in-domain-compromise/
+    - https://www.pingcastle.com/documentation/scanner/
+author: Nasreddine Bencherchali (Nextron Systems), X__Junior (Nextron Systems)
+date: 2024/01/11
+tags:
+    - attack.execution
+    - attack.t1059
+    - attack.defense_evasion
+    - attack.t1202
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        OriginalFileName:
```

Comment:
We could add relevant command lines as well to detect stripped binaries. As long as we don't expect the command lines to appear elsewhere.